src/common/Header/styled.js

```diff
@@ -0,0 +1,154 @@
+import styled from "styled-components";
+import { ReactComponent as VideoSVG } from "../Header/Icons/video.svg";
+import { ReactComponent as SearchSVG } from "../Header/Icons/search.svg";
+
+export const Wrapper = styled.header`
+  max-width: 1920px;
+  margin: 0 auto;
+  background-color: #18181b;
+  color: white;
+  margin-bottom: 58px;
+`;
+
+export const Content = styled.div`
+  max-width: 1260px;
+  margin: 0 auto;
+  padding: 23px;
+  display: flex;
+  justify-content: space-between;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    padding: 16px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    flex-direction: column;
+  }
+`;
+
+export const Container = styled.div`
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-gap: 80px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    grid-gap: 0;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    justify-content: space-between;
+    margin-bottom: 20px;
+  }
+`;
+
+export const Title = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const Link = styled.a`
+  display: flex;
+  align-items: center;
+  font-size: 24px;
+  font-weight: 500;
+  cursor: pointer;
+  letter-spacing: -1.5px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    font-size: 13px;
+    margin-right: 19px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    justify-content: space-between;
+  }
+`;
+
+export const VideoIcon = styled(VideoSVG)`
+  width: 40px;
+  height: 40px;
+  margin-right: 16px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    margin-right: 8px;
+    width: 17px;
+    height: 17px;
+  }
+`;
+
+export const Navigation = styled.nav`
+  display: flex;
+  align-items: center;
+`;
+
+export const List = styled.ul`
+  list-style: none;
+  padding-left: 0px;
+  margin: 0;
+  display: flex;
+`;
+
+export const Item = styled.li`
+  margin-right: 16px;
+  cursor: pointer;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    margin-right: 12px;
+  }
+`;
+
+export const Name = styled.a`
+  font-size: 14px;
+  padding: 8px 24px;
+  text-transform: uppercase;
+
+  &:hover {
+    border: 1px solid white;
+    margin: -1px;
```

Comment:
Maybe changing "border" to "outline" would eliminate need to set margin to -1?

Reply:
Thanks for the suggestion. I checked it and it's a better solution 👍